rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -15,7 +15,39 @@ index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "User Added to Privileged Group in Active Directory"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating User Added to Privileged Group in Active Directory
+
+Privileged accounts and groups in Active Directory are those to which powerful rights, privileges, and permissions are
+granted that allow them to perform nearly any action in Active Directory and on domain-joined systems.
+
+Attackers can add users to privileged groups to maintain their level of access if other privileged accounts they do have
+access to get uncovered by the security team. This way, they can continue their operations even after the discovery of
+an abused account.
```

Comment:
Made some minor changes to remove passive voice and some extra words. Feel free to ignore if this detracts from the original meaning:
Attackers can add users to privileged groups to maintain a level of access if their other privileged accounts are uncovered by the security team. This allows them to keep operating after the security team discovers abused accounts.